README.md

```diff
@@ -145,7 +145,7 @@ Happy hacking!
 [zip-download]: https://cdn.jsdelivr.net/gh/julmot/datatables.mark.js/dist/
 [npm]: https://www.npmjs.com/
 [bower]: https://bower.io/
-[markjs-jsdelivr]: https://cdn.jsdelivr.net/gh/julmot/datatables.mark.js/dist/
```

Comment:
@LukasDrgon this was already the correct link but let's switch it to https://www.jsdelivr.com/package/npm/datatables.mark.js (npm instead of GitHub)

Reply:
Is the link now correctly pointing to the npm version?

Reply:
@julmot the one I left in the comment above is correct for npm. The `/gh/` link works fine too but we recommend always using only one of these methods for every package, with npm being the preferred one (using only npm means you can get detailed package usage statistics and npm packages are also searchable on our website).